runway/model.py

```diff
@@ -86,11 +121,11 @@ def command_route(command_name):
                 for out in outputs:
                     name = out.to_dict()['name']
                     serialized_outputs[name] = out.serialize(results[name])
-                return json.dumps(serialized_outputs).encode('utf8')
+                return jsonify(json.loads(json.dumps(serialized_outputs).encode('utf8')))
```

Comment:
This is a horribly inefficient hack that I'd love an alternative too. I did this for lack of understanding as to why the JSON string `json.dumps(serialized_outputs).encode('utf8')` must be UTF-8 encoded. For fear of breaking existing functionality I forced this by encoding `serialized_outputs` first as a JSON string with UTF-8 encoding, then back to JSON, then back to a string via `jsonify()`. This is gross and we should change it.

Reply:
It would seem that Flask's `jsonify` _may_ be doing unicode encoding on its own (although not sure if its UTF-8 unicode). It looks like its doing it through [itsdangerous's json module](https://github.com/pallets/itsdangerous/blob/master/src/itsdangerous/_json.py) which may or may not use [simplejson](https://simplejson.readthedocs.io/en/latest/) which may or may not have different behaviors with Python 2 and 3... 🤷🏼‍♂️

Reply:
Feel free to remove enforcing the UTF-8 encoding, I forget why we were encoding to UTF-8 but suspect it was a workaround some frontend issue - still, it shouldn't be the responsibility of the model server (or at least it would be specified at the data type level, e.g. this is a UTF-8 text data type).

Reply:
👌 removed.